starknet/block.go

```diff
@@ -14,6 +14,9 @@ type Block struct {
 	StateRoot             *felt.Felt            `json:"state_root"`
 	TransactionCommitment *felt.Felt            `json:"transaction_commitment"`
 	EventCommitment       *felt.Felt            `json:"event_commitment"`
+	ReceiptCommitment     *felt.Felt            `json:"receipt_commitment"`
+	StateDiffcommitment   *felt.Felt            `json:"state_diff_commitment"`
+	StateDiffLength       uint64                `json:"state_diff_length"`
```

Comment:
are these fields for tests only?

Reply:
They seem to be in the block header now, eg
https://integration-sepolia.starknet.io/feeder_gateway/get_block?blockNumber=38748

Reply:
Although the pre-release notes don't indicate that these fields have been added?

Reply:
> They seem to be in the block header now, eg

yep, but we use them in tests only. I haven't pay much attention to it, but old fields (`TransactionCommitment` and `EventCommitment`) are used in tests as well

Reply:
I did this for consistency as was the case for 0.13.1